returnn/tf/updater.py

```diff
@@ -926,9 +942,11 @@ def get_apply_grads_op(self, loss, var_list):
     return tf.group(*all_apply_grads)
 
 
-class KerasOptimizer(Optimizer):
+class KerasOptimizerWrapper(Optimizer):
```

Comment:
Maybe the class name should also reflect that this is for internal use only, by putting an underscore.
```suggestion
class _KerasOptimizerWrapper(Optimizer):
```